src/core/utils/components/VirtualScroll.test.tsx

```diff
@@ -0,0 +1,92 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import { fireEvent, render } from '@testing-library/react';
+import React from 'react';
+import VirtualScroll from './VirtualScroll';
+
+const component = (
```

Comment:
Why you need to have it as a separate variable?

Reply:
removed